src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCaseAboutToSubmitHandlerTest.java

```diff
@@ -35,110 +57,178 @@ public class AdjournCaseAboutToSubmitHandlerTest {
     @Mock
     private CaseDetails<SscsCaseData> caseDetails;
 
+    @Mock
+    private ListAssistHearingMessageHelper hearingMessageHelper;
+
+    @SuppressWarnings("unused")
+    @Mock
     private PreviewDocumentService previewDocumentService;
+
     private SscsCaseData sscsCaseData;
+    private AutoCloseable autoCloseable;
 
     @Before
     public void setUp() {
-        openMocks(this);
-        previewDocumentService = new PreviewDocumentService();
-        handler = new AdjournCaseAboutToSubmitHandler(previewDocumentService);
+        autoCloseable = openMocks(this);
```

Comment:
If you upgrade this class to use jypiter, you can remove this and the `@RunWith(JUnitParamsRunner.class)` which would be good

Reply:
That annotation is already there - what change do you mean?

Reply:
done

Reply:
Great!